src/metabase/routes.clj

```diff
@@ -7,19 +7,22 @@
             [stencil.core :as stencil]
             [metabase.api.routes :as api]
             [metabase.public-settings :as public-settings]
-            [metabase.util :as u]))
+            [metabase.util :as u]
+            [metabase.util.embed :refer [embed-head]]))
```

Comment:
It's probably better to avoid `:refer` when possible, I would just `:require` this `:as embed-util` and do `embed-util/embed-head`